src/types.js

```diff
@@ -356,6 +356,8 @@ export type ReactionType = {
 
 export type LocalizableText = string | { text: string, values: Object };
 
+export type MutedTopic = [];
```

Comment:
We can specify this at more detail 
`export type MutedTopic = [string, string]`

Reply:
Done 👍